WordPress/src/main/java/org/wordpress/android/ui/pages/SnackbarMessageHolder.kt

```diff
@@ -2,9 +2,12 @@ package org.wordpress.android.ui.pages
 
 import androidx.annotation.StringRes
 
+const val INVALID_MESSAGE_RES = -1
```

Comment:
I'm wondering if we could use UIString instead of using an invalid id, wdyt?

Reply:
Converting to UIString looks like a great idea! I'll create a separate PR as it will impact several classes.
I'll convert both messageRes and buttonTitleRes to UIString.

Reply:
Converted here: #12740